bin/generate_kong_config

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+. bin/util
+
+announce "Bundling OpenAPI spec"
+bin/bundle_spec
+
+# a Kong declarative configuration requires a defined target server
+# using http://host.docker.interal allows a containerized Kong Gateway to target
+# a local Conjur instance
+echo "servers:
+  - url: http://host.docker.internal:80" >> spec.yml
+
+announce "Generating Kong declarative configuration yaml"
+docker run --rm -it \
+    -v ${PWD}:/opt/openapi \
+    node:latest \
+    bash -c "
+        npm i -g insomnia-inso \
+        && cd /opt/openapi \
+        && inso --verbose \
+            generate config spec.yml \
+            --type declarative \
+            --output out/kong/kong.yml
+    "
+
+announce "Remove authentication plugins from API routes"
```

Comment:
can you give some more context for what this is / why you're doing it? on the surface looks sus

Reply:
Sure - Kong has a bunch of custom authentication plugins, so when `insmnia-inso` converts the openapi spec to a Kong config, it assumes that the listed authentication schemes are the Kong authentication plugins you want applied to each route, which ends up consuming the `Authorization` header in API calls. There are alternatives but removing Kong auth and rolling with Conjur auth just seemed like the simplest option.

Reply:
oh ha I meant in the announcements when you run the script / in the example README. but thank you :)

Reply:
This was partially so I could remember what to say after the weekend 😆 didn't make it seem that way in my response tho